SEED-platform/seed
# Add canonical water fields

#### Any background context you want to provide?
[ENERGY STAR Portfolio Manager water use docuemtation](https://portfoliomanager.energystar.gov/pm/glossary)

#### What's this PR do?
- Adds canonical Columns associated with PropertyState: Indoor Water Use, Indoor WUI, Outdoor Water Use, Water Use, WUI. Water Use units are `kgal`, and WUI units are `kgal/ft**2/year`. The frontend has been updated to recognize these data types.

- Updates the pm-default-mappings file for these new fields.
- Fixes a few bugs preventing GHG and GHG_I data types from being recognized on certain pages.

#### Questions
- Are the units of water use `kgal` or `kgal/year`
- Should there be additional units for water use (`kgal`) and wui (`kgal/ft**2/year`)? `L/m**2/year`? `gal/ft**2/year`?

#### How should this be manually tested?
Upload a spreadsheet with all known pint data types
- area
- eui
- ghg
- ghg_intensity
- water_use (Indoor Water Use, Outdoor Water Use, and Water Use)
- wui (Indoor WUI, and WUI)
During the mapping step, confirm that the default units are correct

#### What are the relevant tickets?
#4662

#### Screenshots (if appropriate)
Data Upload - Mapping
<img width="797" alt="Screenshot 2024-07-12 at 9 45 26 AM" src="https://github.com/user-attachments/assets/e589ad18-4fa0-44e4-a7c1-2b7c6d89e278">
Inventory List
<img width="1228" alt="Screenshot 2024-07-12 at 9 51 04 AM" src="https://github.com/user-attachments/assets/3d0d98cc-2f7f-45ff-a7d8-dc72e6368b4e">
Column Mapping
<img width="915" alt="Screenshot 2024-07-12 at 9 44 57 AM" src="https://github.com/user-attachments/assets/e7d16d7e-887b-40a1-9dd2-c74da8f3c3f5">
